pkg/assembler/backends/arangodb/certifyLegal_test.go

```diff
@@ -0,0 +1,759 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integration
+
+package arangodb
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/guacsec/guac/internal/testing/ptrfrom"
+	"github.com/guacsec/guac/pkg/assembler/graphql/model"
+)
+
+var t2 = time.Unix(1e9, 0)
+var t3 = time.Unix(1e9+5, 0)
+
+var p1 = &model.PkgInputSpec{
```

Comment:
All the testdata can be found here: https://github.com/guacsec/guac/blob/main/internal/testing/testdata/models.go. Arango and Ent both use this in unit tests

Reply:
Done.